handlers.go

```diff
@@ -251,16 +251,18 @@ func (a *autographer) handleHeartbeat(w http.ResponseWriter, r *http.Request) {
 		result         = map[string]bool{}
 		status         = http.StatusOK
 		requestContext = r.Context()
+		rid            = getRequestID(r)
 	)
 
 	// try to fetch the private key from the HSM for the first
 	// signer conf with a non-PEM private key that we saved on
 	// server start
 	if a.heartbeatConf.hsmSignerConf != nil {
 		var (
-			err         error
-			conf        = a.heartbeatConf.hsmSignerConf
-			checkResult = make(chan error, 1)
+			err                 error
+			conf                = a.heartbeatConf.hsmSignerConf
```

Comment:
nit: `conf` may be used elsewhere so perhaps rename this `hsmConf` to prevent namespace confusion?